nmostesting/NMOSTesting.py

```diff
@@ -660,10 +684,10 @@ def log(self, type, message, *args):
 
 def start_web_servers():
     ctx = None
-    if ENABLE_HTTPS:
+    if CONFIG.ENABLE_HTTPS:

```

Comment:
What happens if this config is changed later? Do we need to stop/start the web servers?

Reply:
Caveats added to README, so can revisit later if demand